.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+name: CI
+on: [push, pull_request]
```

Comment:
* Could you add a cron too, like https://github.com/jolicode/docker-starter/blob/main/.github/workflows/ci.yml#L8-L10
* we will only build push on master and pr: https://github.com/jolicode/docker-starter/blob/main/.github/workflows/ci.yml#L4-L7

(I contribute to this lib via this repo and not my fork. So there is no need to trigger the workflow twice)